README.md

```diff
@@ -15,51 +15,105 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l)== 0:
+    return None
+  return max(l)
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
-# copy & paste your solution here
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+for i in iter:
+    l.append(i)
+  return(l)
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  # a list to hold the list
+  aList = []
```

Comment:
I would reconsider the naming here! If another programmer picked this up, how quickly would they be able to identify which list represents a matrix and which represents a row? Probably not too quickly! `matrix` and `col` would probably be better variable names here.